bin/reth/src/commands/db/diff.rs

```diff
@@ -85,8 +85,8 @@ impl Command {
 
 /// Find diffs for a table, then analyzing the result
 fn find_diffs<T: Table>(
-    primary_tx: impl DbTx,
-    secondary_tx: impl DbTx,
+    primary_tx: &impl DbTx,
+    secondary_tx: &impl DbTx,
```

Comment:
I'm fine with this for the perf improvement, and imo the flexibility of having `impl T` instead of `&impl T`, may break some public api though, so anywhere we should allow this @mattsse ?

Reply:
&impl looks very horrible